docs/release_notes.md

```diff
@@ -4,11 +4,15 @@
 
 ### Notable changes
 
-- Root certificate rotation is now supported through the new CRD, MeshRootCertificate.
+- OSM certificate provider is now configured using the new CRD, MeshRootCertificate
+  - Custom trust domains (i.e. certificate CommonNames) are now supported
+- The authentication token used to configure the Hashicorp Vault certificate provider can now be passed in using a secretRef
```

Comment:
Thanks for making these updates!

Reply:
Agreed, waiting on @steeling to confirm

Reply:
the custom trust domain is only plumbed through on MRC's. it would take a small amount of work to add the flag to the binary and pass to the cert manager. have we cut a release yet? if not i can do for tomorrow

Reply:
Yeah no release has been cut yet right @nojnhuh

Reply:
Correct, I was going to make sure this was merged before cutting the release.

Reply:
done now with https://github.com/openservicemesh/osm/pull/4877